.azure-pipelines/stages.yml

```diff
@@ -403,6 +403,7 @@ stages:
       parameters:
         managedAgent: false
         ciTarget: bazel.release
+        bazelBuildExtraOptions: "--sandbox_base=/tmp/sandbox_base"
```

Comment:
this is added because the x64 build job now uses a self-hosted agent and so has a lot more memory and can do this - it brings it into line with other jobs

for this the better solution is probably that this is added elsewhere depending on the capability etc (there are other moving parts here)

atm im more focused on regorganizing things that doccing them - mostly i want to get a clearer separation of responsibilities in CI - its a bit confused/confusing atm

Reply:
sure I just think it'd be helpful t have more explanation of what these do inline.